src/MSALWrapper/AuthFlow/Broker.cs

```diff
@@ -23,6 +24,7 @@ public class Broker : IAuthFlow
         private readonly string promptHint;
         private readonly IList<Exception> errors;
         private IPCAWrapper pcaWrapper;
+        private int interactivePromptsCount;

```

Comment:
Discussed offline - because we do not have a specific question to answer by tracking prompt count, we don't need to collect this information right now. Generally speaking a run of an auth flow should have 0 (silent) or 1 (interactive) prompts and 1 is already more than desired. So we're going to remove this count.